pkg/promtail/targets/filetarget.go

```diff
@@ -304,7 +306,16 @@ func (t *FileTarget) stopTailing(ps []string) {
 func toStopTailing(nt []string, et map[string]*tailer) []string {
 	// Make a set of all existing tails
 	existingTails := make(map[string]struct{}, len(et))
-	for file := range et {
+	removeTails := make([]string, len(et))
+	for file, tailer := range et {
+		fileStat, err := os.Stat(file)
+		if err != nil {
+			//level.Error(t.logger).Log("msg", "failed to get file stats", "file", file.path, "error", err)
+		}
+		pos, _ := tailer.positions.Get(file)
+		if time.Now().Sub(fileStat.ModTime()) > 10*time.Minute && (pos == fileStat.Size()) {
```

Comment:
I don't see config getting used here. The waiting period is hardcoded instead.
Can you please fix it?

Reply:
Hey @sandeepsukhani. Thanks for the review.
Yeah, this is hard coded. This was just a test. Will clean this up